pkg/plugins/manager/loader/assetpath/assetpath.go

```diff
@@ -153,3 +153,22 @@ func getBaseDir(pluginDir string) string {
 	}
 	return baseDir
 }
+
+func (s *Service) GetTranslations(n PluginInfo) (map[string]string, error) {
+	pathToLocales, err := s.RelativeURL(n, "locales/")
```

Comment:
nit: does this needs the trailing slash? I don't think so because it's later used with `url.JoinPath`. If that's the case I'd remove it.

Also, should this be `pathToTranslations`? Just to keep the naming style.

Reply:
Good catch, I'll remove the trailing slash and fix the naming 👍

Reply:
Fixed 👍